docs/source/download.rst

```diff
@@ -1,64 +1,103 @@
-=============
-Obtain Devito
-=============
-
-There are three main approaches to installing Devito. For those looking for the
-least-friction way to try Devito, we recommend the Docker route. For those
-looking to use Devito alongside other packages as part of a project, we support
-pip installation. If you're looking to develop for Devito, you might benefit from
-using the included conda environment that includes all the bells and whistles we
+===========================
+Download and Install Devito
+===========================
+
+There are three main approaches to installing Devito.
+
+For those looking for a quick “out-of-the-box” way to try Devito, we recommend
+the route of `Docker installation`_.
+For those interested in running on CPUs, please follow `Docker for CPUs`_.
+For those interested in deploying on GPUs, please follow `Docker for GPUs`_.
+For those looking to use Devito alongside other packages as part of a project, we support
+`pip installation`_. If you are looking to develop for Devito, you might benefit from
+following the `conda installation`_  that includes all the bells and whistles we
 recommend when developing for Devito.
 
+
 Docker installation
 -------------------
+
 You can install and run Devito via Docker_:
 
 .. _Docker: https://www.docker.com/  
 
+Docker for CPUs
+```````````````
+
 .. code-block:: shell
 
-   # get the code
-   git clone https://github.com/devitocodes/devito.git
-   cd devito
+   # NOTE (admin-sudo access may be needed) for some steps
+
+   # 1. Pull Devito image
+   docker pull devitocodes/devito:cpu-latest
 
-   # 1. run the tests
-   docker-compose run devito /tests
+   # 2. (Optional but recommended) Run the tests
+   docker run --rm --name testrun 'devitocodes/devito:cpu-latest' pytest tests/test_operator.py
```

Comment:
what's basically the difference between this and the old docker-compose way?

Reply:
well, if you're changing it, maybe we should find out first

Reply:
Docker compose is useful for people who want to develop Devito while using it inside Docker. It will rebuild images locally, while this one downloads images from docker hub. 

So maybe keep the docker compose as a developer option. 

@FabioLuporini the specialist configuration we use (not in this approach, but in the docker-compose approach) is around mounting directories into the container so the code can be essentially "live-edited", and for mapping ports corresponding to Jupyter notebooks.

Reply:
hmm I see... indeed someone may want to do this...will add something